pynest/nest/lib/hl_api_nodes.py

```diff
@@ -115,10 +124,10 @@ def Create(model, n=1, params=None, positions=None):
 
     node_ids = spp()
 
-    if params is not None and params_contains_list:
+    if params is not None and iterable_or_parameter_in_params:
         try:
             SetStatus(node_ids, params)
-        except:
+        except Exception:
```

Comment:
don't we have a more specific exception to catch here? Seems overly inclusive to only write a warning when anything and everything could have gone wrong.

Reply:
The error is still raised:
https://github.com/nest/nest-simulator/blob/e287301bd5e0c2ff68e7391e00932a114033aca0/pynest/nest/lib/hl_api_nodes.py#L130-L134

So scripts will exit with an error if something goes wrong. But for interactive work I suppose it can be useful to know that the nodes have been created.

Are there any exceptions for which we don't want to issue a warning if an exception is raised here?

Reply:
You are right. It makes sense to raise a warning and pass the error on.
Currently I can't think of a type of exception where this warning would be overly harmful.
:+1: